staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -22,6 +22,9 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/klog"
```

Comment:
nit: previously it was sorted alphabetically, so I think it made more sense previously

Reply:
Thanks. Looks like I still don't have good import settings in my IDE. I'll work on that.  Fixed!